build/sbt

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 
-SELF=$(cd $(dirname $0) && pwd)
+SELF=$(cd "$(dirname "$0")" && pwd)
```

Comment:
Are these changes necessary/related?

Reply:
This is shellcheck error/warning fix. I think we can keep it.